compiler/src/iree/compiler/Codegen/Passes.h

```diff
@@ -521,6 +521,10 @@ std::unique_ptr<OperationPass<func::FuncOp>> createSPIRVVectorizePass();
 /// having pointer bitcast.
 std::unique_ptr<OperationPass<ModuleOp>> createSPIRVVectorizeLoadStore();
 
+/// Breaks down large vendors not natively supported by SPIR-V.
```

Comment:
s/vendors/vectors/

Reply:
could we break down large vendors that dont natively support SPIR-V. That would be really good result! :D

Reply:
lol. Silly typo by me. Fixed.